format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -282,6 +268,112 @@ object SyntaxTokens {
     def tokensWith: KwWith = tree.findBetween[KwWith](_.lhs, _.rhs).get
   }
 
+  // == Defn ==
+  implicit class XtensionDefnClassSyntax(val tree: Defn.Class) extends AnyVal {
+    def tokensClass: KwClass = tree.find[KwClass].get
```

Comment:
Counter example
```
@Foo({
  class Bar
  new Bar
})
class Banana()
```
It's better to go in reverse order from the name.

Reply:
merged too quickly: https://github.com/scalacenter/scala-syntax/pull/35